src/main/kotlin/com/autonomousapps/tasks/ProjectHealth.kt

```diff
@@ -0,0 +1,32 @@
+package com.autonomousapps.tasks
+
+import com.autonomousapps.TASK_GROUP_DEP
+import com.autonomousapps.advice.ComprehensiveAdvice
+import com.autonomousapps.internal.utils.fromJson
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
+import org.gradle.api.tasks.TaskAction
+
+abstract class ProjectHealth : DefaultTask() {
+
+  init {
+    group = TASK_GROUP_DEP
+    description = "Consumes output of aggregateAdvice and can fail the build if desired"
```

Comment:
I would suggest to re-phrase so the description is end-user facing rather than internal to plugin.

For instance: `Generates report on the dependencies and plugins (api vs implementation, unsed dependencies, ...)`